Themis/Views/Assessment/CodeEditor/CodeView.swift

```diff
@@ -59,7 +59,6 @@ struct CodeView: View {
     var editorFlags: CodeEditor.Flags {
         var flags: CodeEditor.Flags = []
         flags.insert(.selectable)
-        if colorScheme == .dark { flags.insert(.blackBackground) }
```

Comment:
Why is this no longer necessary? I think I remember we added something like this because the background of the theme we chose was too grey for our taste. Has something changed there?